Gulpfile.js

```diff
@@ -1057,51 +1105,6 @@ gulp.task("build-rules", "Compiles tslint rules to js", () => {
         .pipe(gulp.dest(dest));
 });
 
-const lintTargets = [

```

Comment:
This chunk of code was unused (we swapped to shelling out to tslint directly once they added project mode), so I removed it while I was here.